### PR TITLE
created Book model + repository and service layers

### PR DESCRIPTION
**Pull request by validatorr**

### CODE REVIEW

**Review by don-bigdad — APPROVED** [review]
Great job!

File: src/main/java/v/hryhoryk/onlinebookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
```
Comment (Andriy-dz) [inline src/main/java/v/hryhoryk/onlinebookstore/model/Book.java:15]: add table annotation

File: src/main/java/v/hryhoryk/onlinebookstore/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
```
Comment (Andriy-dz) [inline src/main/java/v/hryhoryk/onlinebookstore/model/Book.java:27]: conditions are missing above the fields (unique, not null)

**Review by Andriy-dz — CHANGES_REQUESTED** [review]
Good Job! Lets improve your solution

File: src/main/java/v/hryhoryk/onlinebookstore/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table
```
Comment (Elena-Bruyako) [inline src/main/java/v/hryhoryk/onlinebookstore/model/Book.java]: Let's add the name of table

File: src/main/java/v/hryhoryk/onlinebookstore/service/BookServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import v.hryhoryk.onlinebookstore.model.Book;
+import v.hryhoryk.onlinebookstore.repository.BookRepository;
+
+@Service
+public class BookServiceImpl implements BookService {
+
```
Comment (Elena-Bruyako) [inline src/main/java/v/hryhoryk/onlinebookstore/service/BookServiceImpl.java]: remove empty line

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Let's fix some comments.

**Review by Elena-Bruyako — APPROVED** [review]

**Pull request merged**